cmd/kubelet/app/options/options.go

```diff
@@ -498,6 +498,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Int32Var(&c.KubeAPIQPS, "kube-api-qps", c.KubeAPIQPS, "QPS to use while talking with kubernetes apiserver. The number must be >= 0. If 0 will use DefaultQPS: 5. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags")
 	fs.Int32Var(&c.KubeAPIBurst, "kube-api-burst", c.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver. The number must be >= 0. If 0 will use DefaultBurst: 10. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags")
```

Comment:
Should we deprecate these two params and suggest users use MaxParallelImagePulls in the future?

Reply:
I guess you mean RegistryPullQPS and RegistryBurst. 

Yeah, I cannot find the rationale behind why they were added in the first place. We should discourage people from using them and encourage using maxParallelImagePulls once maxParallelImagePulls goes Beta, and then remove these two after maxParallelImagePulls goes GA

Reply:
Can you create a bug to deprecate these two? We can use bug to collect feedback.

Reply:
> Can you create a bug to deprecate these two? We can use bug to collect feedback.

@SergeyKanzhelev  https://github.com/kubernetes/kubernetes/issues/112044 this one was opened before and maybe we can use this one to track it.